torchbenchmark/util/backends/torchdynamo.py

```diff
@@ -24,10 +26,18 @@ def parse_torchdynamo_args(model: 'torchbenchmark.util.model.BenchmarkModel', dy
 
 
 def apply_torchdynamo_args(model: 'torchbenchmark.util.model.BenchmarkModel', args: argparse.Namespace, precision: str):
+    torchdynamo.config.raise_on_backend_error = False
```

Comment:
Why is the function modified?

Reply:
torchdynamo.config.raise_on_backend_error means throw except when a subgraph compiling error. It would be a nice way to just skip this subgraph.